component-samples/map-components/samples/angular/src/app/app.component.css

```diff
@@ -1,9 +1,6 @@
-@import url('https://js.arcgis.com/4.30/@arcgis/core/assets/esri/themes/dark/main.css');
+@import url('https://js.arcgis.com/4.31/@arcgis/core/assets/esri/themes/dark/main.css');
 
 arcgis-map {
-  width: 100%;
-  height: 100%;
-  position: absolute;
-  top: 0;
-  left: 0;
+  display: block;
```

Comment:
Double check this works. I think you'll also need the following CSS in /app/src/styles.css:

```css

html,
body {
  padding: 0;
  margin: 0;
  height: 100%;
  width: 100%;
}
```